src/main/java/org/sagebionetworks/bridge/worker/ExportWorker.java

```diff
@@ -0,0 +1,69 @@
+package org.sagebionetworks.bridge.worker;
+
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * Generic export worker. This class encapsulates basic configuration, helpers, and clients needed by export workers
+ * and serves as the parent class for both Synapse export workers and iOS survey export workers.
+ */
+public abstract class ExportWorker extends Thread {
+    // Configured externally
+    private ExportWorkerManager manager;
+    private String studyId;
+
+    // Internal state
+    private final LinkedBlockingQueue<ExportTask> taskQueue = new LinkedBlockingQueue<>();
```

Comment:
Probably want to limit the size of the queue so you don't run out of memory if takers are slow